File: Java/4th Sem/Day2/q3.java
public class q3 {
    public static void main(String[] args) {
        int arr[]={15,20,13,24,53,6,71,8,39,10};
        for(int i=0;i<9;i++){
            for(int j=i+1;j<10;j++){
                if(arr[i]>arr[j]){
                    int temp=arr[i];
                    arr[i]=arr[j];
                    arr[j]=temp;
                }
            }
        }
        for(int i=0;i<10;i++){
            System.out.print(arr[i]);
            System.out.print(" ");
        }
    }
    
}

File: 5th sem/day1/q3.c
//number of duplicates in an array and most repeated element and how many times it is repeated
#include <stdio.h>
int main() {
    int n;
    printf("Enter the size of the array: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter the elements of the array:\n");
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int count = 0;
    int maxCount = 0;
    int maxElement = arr[0];

    for (int i = 0; i < n; i++) {
        for (int j = i + 1; j < n; j++) {
            if (arr[i] == arr[j]) {
                count++;
                if (count > maxCount) {
                    maxCount = count;
                    maxElement = arr[i];
                }
            }
        }
        count = 0;
    }

    printf("Number of duplicates: %d\n", maxCount);
    printf("Most repeated element: %d\n", maxElement);
    printf("Number of times repeated: %d\n", maxCount);

    return 0;
}
